app/src/main/java/com/fstyle/androidtrainning/screen/main/MainActivity.java

```diff
@@ -1,33 +1,50 @@
 package com.fstyle.androidtrainning.screen.main;
 
+import android.content.BroadcastReceiver;
+import android.content.Context;
 import android.content.Intent;
+import android.content.IntentFilter;
 import android.os.Bundle;
 import android.support.annotation.IntDef;
 import android.support.v4.view.ViewPager;
 import android.support.v7.widget.SearchView;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
 import com.fstyle.androidtrainning.R;
 import com.fstyle.androidtrainning.application.MainApplication;
 import com.fstyle.androidtrainning.data.remote.service.config.LastFmApi;
+import com.fstyle.androidtrainning.model.Track;
 import com.fstyle.androidtrainning.screen.BaseActivity;
+import com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong.ListSongAdapter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports